x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -316,6 +317,13 @@ export const AddExceptionModal = memo(function AddExceptionModal({
     [fetchOrCreateListError, exceptionItemsToAdd]
   );
 
+  const isRuleEQLSequenceStatement = useMemo((): boolean => {
+    if (maybeRule != null && maybeRule.query != null) {
+      return maybeRule.type === 'eql' && maybeRule.query.startsWith('sequence');
```

Comment:
This is the logic I chose to go with for delineating sequence queries and non-sequence queries. When rylands validation pr gets merged, this should cover all cases given that sequence is a saved term in eql, but any comments/suggestions from those more versed in EQL would be appreciated

Reply:
@rw-access may have ideas about the best way to do this

Reply:
@dplumlee FYI we have [some helpers](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/detection_engine/utils.ts#L20) available to avoid these kinds of hardcoded strings. Since this logic is duplicated in a few spots, I'd say that moving this function out to a shared location makes sense.

Reply:
is `sequence where ...` valid eql? I tested using `sequence` as an event category in the EQL ES api and got an exception, wasn't sure if that just isn't allowed since `sequence` is a reserved word.

Reply:
I haven't tried `sequence where true` in ES EQL, wasn't sure how ANTLR handled it.